### PR TITLE
Some map stuff affecting tram/delta

### PR DESCRIPTION
**Pull request by jughu**
Adds a missing xmas tree to tramstation (located in the chapel because i could find a nice place in bar)
Removes seclite that was mapped in deltastation and adds it in the locker itself instead like all other security lockers does
adds a contrabrand and armoury spawner to deltastation armour if you think armoury spawner is too much and too streamlining i can always remove it and just keep a single contrabrand spawner

:cl: Improvedname
fix: streamlines deltastation with other maps and adds a contrabrand/armoury spawner to the locker
balance: Removes seclite from deltastation, and inserted it in the basic sec locker kit like all other officers have
fix: Adds a xmas tree to tramstation chapel
/:cl:

### CODE REVIEW

**Comment by Sealed101** [comment]
wait lmao did tram not have a christmas tree again

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -62,6 +62,7 @@
 	new /obj/item/storage/box/flashbangs(src)
 	new /obj/item/shield/riot/tele(src)
 	new /obj/item/storage/belt/security/full(src)
+	new /obj/item/flashlight/seclite(src)
```
Comment (san7890) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:65]: This is a balance change, not a fix, because now it'll apply to all five maps instead of just Tram. Either revert this change or remark your chanelog.

Comment (jughu) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm:65]: done

**Comment by san7890** [comment]
your GBP is too low for me to accommodate allowing in the balance change. feel free to remake this PR when you get out of the ditch you're in.

in the future, please atomicize your PRs. bundling in the tramstation tree fix along with balance changes to security on all maps, as well as some other delta things is what delays PRs like these. when feasible, one change should be met with one fix PR (i.e.: Tramstation Tree Fix, DeltaStation Armory Fix, etc.) Please do re-open these as new individual PRs so they can be merged on their own merits.
